docs/pages/install/docker.mdx

```diff
@@ -11,7 +11,7 @@ Pelican builds separate images for each Pelican server components, e.g. origin,
 * Pelican director server: `hub.opensciencegrid.org/pelican_platform/director:latest`
 * Pelican registry server: `hub.opensciencegrid.org/pelican_platform/registry:latest`
 
-> Note: the `latest` tag will pull the Pelican server image with the latest released version. You may pull an image with explicit Pelican version by passing the version as the tag (e.g. `v7.6.4`) instead. For a list of available tags, refer to [Pelican Harbor repository](https://hub.opensciencegrid.org/harbor/projects/883/repositories/origin/artifacts-tab?publicAndNotLogged=yes).
+> **Note**: the `latest` tag will pull the Pelican server image with the latest released version. You may pull an image with explicit Pelican version by passing the version as the tag (e.g. `v7.6.4`) instead. For a list of available tags, refer to [Pelican Harbor repository](https://hub.opensciencegrid.org/harbor/projects/883/repositories/origin/artifacts-tab?publicAndNotLogged=yes).
```

Comment:
This Harbor URL brings users to a page with a big scary red banner that says "Unauthorized". Presumably we have an image for every tagged/released version, so maybe we can point them to the github Releases page instead?

Reply:
I'm also not a fan of the scary red banner. That probably requires something to change on the Harbor side. 

Maybe link to a document (in Github?) that tables the versions and the Harbor tags / docker addresses? That way people can find the docker addresses without encountering the scary red banner.

Reply:
I'd suggest change the GitHub release note to include links to various Docker images, or have a page on our docs/Pelican website listing available containers. @CannonLock What's your thought on this?